src/main/java/dev/slimevr/bridge/ProtobufBridge.java

```diff
@@ -188,7 +190,9 @@ protected T getInternalRemoteTrackerById(int trackerId) {
 	@VRServerThread
 	protected void reconnected() {
 		for (ShareableTracker tracker : sharedTrackers) {
```

Comment:
Looks like the autoformatter is not forcing tab based indentation, but rather taking whatever indentation style is used in the current file. In this case, this file looks like its using 2 spaces instead of tabs.

IMO the autoformatter should force whatever indentation style that is used to be consistent across all files, not just within a file

Reply:
Wait what? Checking this in the comment view shows it as a tab, but checking it in the "changes" view on github shows it as two spaces. Whats going on?

Reply:
OK, checking out the branch makes it clear that its actually tabs, so I guess that resolves this comment. Super weird it didn't display that way on github.

Feel free to mark as resolved @Eirenliel